docs/topics/clusterdefinitions.md

```diff
@@ -486,6 +455,20 @@ Below is a list of kube-scheduler options that are _not_ currently user-configur
 
 We consider `kubeletConfig`, `controllerManagerConfig`, `apiServerConfig`, and `schedulerConfig` to be generic conveniences that add power/flexibility to cluster deployments. Their usage comes with no operational guarantees! They are manual tuning features that enable low-level configuration of a kubernetes cluster.
 
+#### Custom YAML for Kubernetes component manifests
+
+Custom YAML specifications can be configured for kube-scheduler, kube-controller-manager, cloud-controller-manager and kube-apiserver in addition to the addons described [above](#addons). You will need to pass in a _base64_ encoded string of the kubernetes manifest YAML file to _KubernetesComponentConfig["data"]_ . For example, to pass a custom kube-scheduler config, do the following:
+
+```json
+"kubernetesConfig": {
+    "schedulerConfig": {
+            "data" : "<base64 encoded string of your k8s manifest YAML>"
+        }
+}
+```
+
+CAVEAT: Please note that this is an experimental feature. Since Addons.Data allows you to provide your own scripts, you face the risk of any unintended/undesirable consequences of the errors and failures from running that script.
```

Comment:
Maybe rewrite slightly?

> _**NOTE**_: Custom YAML for addons is an experimental feature. Since `Addons.Data` allows you to provide your own scripts, you are responsible for any undesirable consequences of their errors or failures. Use at your own risk.